ci/release/android_sdk.go

```diff
@@ -48,28 +55,148 @@ func ReleaseAndroidSDK() error {
 		log.Info("not a tag build, skipping ReleaseAndroidSDK action...")
 		return nil
 	}
-	err = env.EnsureEnvVars(
-		ProxySonatypeUsername,
-		ProxySonatypePassword,
-		SonatypeUsername,
-		SonatypePassword,
+
+	err = storage.DownloadArtifacts()
+	if err != nil {
+		return err
+	}
+
+	err = env.EnsureEnvVars(BintrayUsername, BintrayToken)
+	if err != nil {
+		return err
+	}
+	repositoryURL, err := url.Parse("https://api.bintray.com/content/mysteriumnetwork/maven")
+	if err != nil {
+		return err
+	}
+	uploader := newBintrayReleaser(
+		&releaseOpts{
+			groupId:    "network.mysterium",
+			artifactId: "mobile-node",
+			version:    env.Str(env.BuildVersion),
+		},
+		&bintrayOpts{
+			repositoryURL: repositoryURL,
+			username:      env.Str(BintrayUsername),
+			password:      env.Str(BintrayToken),
+		},
 	)
+	if err := uploader.upload("build/package/Mysterium.aar"); err != nil {
+		return err
+	}
+	if err := uploader.upload("build/package/mvn.pom"); err != nil {
+		return err
+	}
+	return uploader.publish()
+}
+
+type releaseOpts struct {
+	groupId, artifactId, version string
+}
+
+type bintrayOpts struct {
+	repositoryURL      *url.URL
+	username, password string
+}
+
+// bintrayReleaser uploads and publishes new versions in Bintray via its REST API: https://bintray.com/docs/api/
+type bintrayReleaser struct {
+	client      *http.Client
+	releaseOpts *releaseOpts
+	bintrayOpts *bintrayOpts
+}
+
+func newBintrayReleaser(releaseOpts *releaseOpts, bintrayOpts *bintrayOpts) *bintrayReleaser {
+	return &bintrayReleaser{
+		client:      &http.Client{Timeout: 30 * time.Second},
+		releaseOpts: releaseOpts,
+		bintrayOpts: bintrayOpts,
+	}
+}
+
+func (up *bintrayReleaser) upload(filename string) error {
+	log.Info("Uploading: ", filename)
+
+	file, err := os.Open(filename)
 	if err != nil {
 		return err
 	}
-	err = storage.DownloadArtifacts()
+	defer file.Close()
+
+	requestURL := uploadURL(*up.releaseOpts, *up.bintrayOpts, filename)
+	log.Info("Upload URL: ", requestURL)
+
+	req, err := http.NewRequest(http.MethodPut, requestURL.String(), file)
+	if err != nil {
+		return err
+	}
+	req.SetBasicAuth(up.bintrayOpts.username, up.bintrayOpts.password)
```

Comment:
could extract the auth + header setting into a separate func since you're doing it more than once